templates/about.html

```diff
@@ -18,6 +18,36 @@ <h2>
 		</p>
 	</section>
 
+	<section id="contribute" class="container">
+		<h2>Contribute</h2>
+
+		<p>
+			The HTTP Archive is an open source project maintained by a core group of developers and contributors from the community. Here's how you can become more involved with the project:
```

Comment:
I propose we..

- (a) link "core group of developers and contributors" to https://httparchive.org/faq#who-maintains-the-http-archive
- (b) update said page to include Paul + restructure that section a little bit by clarifying honorary members vs current maintainers. Something like

> Current core maintainers are X, Y, Z, D. 
> 
> Many people have contributed(link) and helped make HTTP Archive successful over the years. Special thanks to Steve Souders, who started the project in 2010, Pat Meenan who built the WebPageTest infrastructure powering HTTP Archive, and Guy Leech and Stephen Hay for design help along the way.

Reply:
Done

![image](https://user-images.githubusercontent.com/1120896/44599004-e8f96880-a7a2-11e8-8713-7734a1a515f1.png)

![image](https://user-images.githubusercontent.com/1120896/44599017-f0207680-a7a2-11e8-928a-988802865cea.png)